crates/viewer/re_viewer/src/app.rs

```diff
@@ -565,29 +565,36 @@ impl App {
                 store_hub.clear_active_blueprint();
                 egui_ctx.request_repaint(); // Many changes take a frame delay to show up.
             }
-            SystemCommand::UpdateBlueprint(blueprint_id, chunks) => {
+
+            SystemCommand::AppendToStore(store_id, chunks) => {
                 re_log::trace!(
-                    "Update blueprint entities: {}",
+                    "Update entities: {}",
                     chunks.iter().map(|c| c.entity_path()).join(", ")
                 );
```

Comment:
before it was clear where we'd write to, now it would be nice to add which store we're writing to the trace